packages/react-i18n/README.md

```diff
@@ -126,6 +126,7 @@ The provided `i18n` object exposes many useful methods for internationalizing yo
 - `formatCurrency()`: formats a number as a currency according to the locale. Its behaviour depends on the `form:` option.
   - if `form: 'short'` is given, then a possibly-ambiguous short form is used, consisting of the bare symbol if the currency has a symbol, or the ISO 4217 code if there is no symbol for that currency. Examples: `CHF 1,25`, `€ 1,25 EUR`, `OMR 1.250`, `$ 1.25 USD`
   - if `form: 'explicit'` is given, then the result will be the same as for `short`, but will append the ISO 4217 code if it is not already present
+  - if `form: 'auto'` is given, then `explicit` will be selected the `currency` option does not match the `defaultCurrency`, otherwise `short` is selected
```

Comment:
```suggestion
  - if `form: 'auto'` is given, then `explicit` will be selected if the `currency` option does not match the `defaultCurrency`, otherwise `short` is selected
```

Reply:
Do you also want to specify when `currency` or `defaultCurrency` is `undefined`?

Reply:
yea maybe i'll add a secondary statement about undefined values.